script/script.js

```diff
@@ -0,0 +1,199 @@
+            <td class='border border-dark'><div id='cell_7' class='cell d-flex justify-content-center align-items-center'>${board[7]}</div></td>
+            <td class='border border-dark'><div id='cell_8' class='cell d-flex justify-content-center align-items-center'>${board[8]}</div></td>
+          </tr>          
+        </tbody>
+      </table>
+    `;
+
+    document.getElementById('divHeaderGameBoard').innerHTML = "Let's play!";
+    document.getElementById('divGameBoard').innerHTML = divGameBoard;
+  };
+  const reset = () => {
+    for (let i = 0; i < 9; i += 1) {
+      board[i] = ' ';
+    }
+  };
+  return {
+    move, print, empty, winner, reset, draw,
+  };
+})();
+
+const Player = (name, symbol) => {
+  let score = 0;
+  const getName = () => name;
+  const getSymbol = () => symbol;
+  const getScore = () => score;
+  const win = () => {
+    score += 1;
+    return score;
+  };
+  return {
+    getName, getSymbol, getScore, win,
+  };
+};
+
+const GameControl = (() => {
+  const board = Gameboard;
+  let p1 = null;
+  let p2 = null;
+  let currentPlayer = p1;
+  let firstPlayer = p1;
+  let enabled = true;
+  const createPlayers = (p1Name, p2Name) => {
+    p1 = Player(p1Name, '☠️');
+    p2 = Player(p2Name, '🌘');
+  };
+
+  const altScoreBoard = () => {
+    const scoreBoard = `
+      <div class='mb-2 font-weight-bold'>${p1.getName()} V.S. ${p2.getName()}</div>
+      <div class='mb-2'>${p1.getName()}: ${p1.getScore()} victories.</div>
+      <div class='mb-3'>${p2.getName()}: ${p2.getScore()} victories.</div>
+      <button id='btnResetBoard' class='btn btn-primary'>Reset the board</button>
+    `;
+    document.getElementById('divHeaderControls').innerHTML = 'Game Score';
+    document.getElementById('divGameControls').innerHTML = scoreBoard;
+    document.getElementById('btnResetBoard').addEventListener('click', () => {
+      board.reset();
+      for (let i = 0; i < 9; i += 1) {
+        document.getElementById(`cell_${i}`).innerHTML = ' ';
+      }
+      enabled = true;
+      document.getElementById('divHeaderGameBoard').innerHTML = "Let's play!!";
+      document.getElementById('divHeaderGameBoard').className = 'card-header';
+    });
+  };
+
+  const clickEvent = (index) => {
+    if (board.empty(index) && enabled) {
+      board.move(currentPlayer.getSymbol(), index);
+      document.getElementById(`cell_${index}`).innerHTML = currentPlayer.getSymbol();
+      const wins = board.winner(currentPlayer.getSymbol());
+      if (wins) {
+        currentPlayer.win();
+        altScoreBoard();
+        document.getElementById('divHeaderGameBoard').innerHTML = `${currentPlayer.getName()} wins!`;
+        document.getElementById('divHeaderGameBoard').className = 'font-weight-bold text-white card-header bg-success';
+        enabled = false;
+        if (firstPlayer === p1) { firstPlayer = p2; } else { firstPlayer = p1; }
```

Comment:
Opps, everything looks good but we need to work on the code splitting.
Let separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/